_pages/events.md

```diff
@@ -34,7 +34,7 @@ November 20, 2020 - [Brief](https://www.youtube.com/watch?v=XybGruUNc7s&t=5s) to
 Recap of our guest speaker, Miles Tiglao:
 
 > For our non-CAC members, [Zoom Recording](https://us02web.zoom.us/rec/share/8qntZYHsw4H1xkvCFS6YfwuYNu9eweyhi22OUYHwsQJwbFywtCqadBkYzQ_0nsFY.Szee4QhLQhLRYIGP) 
-* Passcocde: Z?E$C28k
+* Passcode: Z?E$C28k
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)

Reply:
Good bot. These are meant to be hard coded, information is public and meant to distribute Zoom recording.